flake.nix

```diff
@@ -105,11 +104,12 @@
         # Run tests with cargo-nextest
         # Consider setting `doCheck = false` on `my-crate` if you do not want
         # the tests to run twice
-        alpacka-nextest = craneLib.cargoNextest {
-          inherit cargoArtifacts src buildInputs;
-          partitions = 1;
-          partitionType = "count";
-        };
+        # Since the tests require internet access, nix can't run them
```

Comment:
Same thing here, I believe we can disable the sandbox altogether. We're ensuring our flake is pure anyways

Reply:
How can we disable the sandbox?

Reply:
The easiest way would be to run our tests with the `--no-sandbox` flag, though we could make it for the whole flake (`nix.useSandbox = false;`) if we wanted to.

Reply:
> The easiest way would be to run our tests with the `--no-sandbox` flag, though we could make it for the whole flake (`nix.useSandbox = false;`) if we wanted to.

I think running the tests with `--no-sandbox` works best. Will do this later